File: fix_search_border.py
import re

def update_search_border(filepath):
    with open(filepath, 'r') as f:
        content = f.read()

    # Old: class="flex-1 ml-5 mr-3 flex items-center h-[44px] border-b border-gray-300 focus-within:border-brand-red transition-colors"
    # New: class="flex-1 ml-5 mr-3 flex items-center h-[44px] border-b-[2px] border-gray-300 focus-within:border-brand-red transition-colors"
    
    content = content.replace(
        'border-b border-gray-300 focus-within:border-brand-red',
        'border-b-[2px] border-gray-300 focus-within:border-brand-red'
    )

    with open(filepath, 'w') as f:
        f.write(content)
    print(f"Updated search border in {filepath}")

update_search_border('/Users/aliceer/UDO/frontend/src/main.js')
update_search_border('/Users/aliceer/UDO/frontend/src/product.js')

File: fix_search_border.js
const fs = require('fs');
let content = fs.readFileSync('frontend/src/main.js', 'utf8');

const target = 'focus-within:border-brand-red/50 focus-within:ring-[3px] focus-within:ring-brand-red/20 transition-all duration-200 relative';
const replacement = 'focus-within:border-[#ff6b6b] focus-within:ring-[1.5px] focus-within:ring-[#ff6b6b] transition-all duration-200 relative';

if (content.includes(target)) {
    content = content.replace(target, replacement);
    fs.writeFileSync('frontend/src/main.js', content, 'utf8');
    console.log("Updated search focus style to a single solid layer.");
} else {
    console.log("Could not find the target text.");
}
